pkg/module/reporter/htmlreporter/htmlreporter.go

```diff
@@ -125,17 +132,23 @@ func (r *HTMLReporter) PostReport() error {
 	if !r.Keep {
 		defer r.cleanup()
 	}
+	workingDir := "/tmp"
+	contentDir := strings.TrimPrefix(r.workingDir, "/tmp/")
```

Comment:
Don't use such specific paths. It is error prone and not portable. The system's base tempdir can be obtained via https://golang.org/pkg/os/#TempDir